go.mod

```diff
@@ -108,11 +108,8 @@ replace (
 	// before removing it make sure that the next version of the related k8s dependencies contains the fix
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.44.0
 
-	// The crypto is pulled from go/compute which is pulled by go/storage
-	// this replace can be removed when version 1.36.1 of go/storage is released.
-	// https://github.com/googleapis/google-cloud-go/tree/main/storage
-	// Fixes CVE-2023-48795 - golang.org/x/crypto Authentication Bypass by Capture-replay
-	golang.org/x/crypto => golang.org/x/crypto v0.17.0
```

Comment:
Did we mean to remove this? I'm seeing the version as `cloud.google.com/go/storage v1.33.0 // indirect`

Reply:
it has been updated, see line 74.

Reply:
ah I see